packages/vbet/vbet/vbet.py

```diff
@@ -85,10 +85,12 @@ def vbet(huc, flowlines, flowareas, orig_slope, max_slope, orig_hand, hillshade,
     vbet_network(flowlines, flowareas, cfg.OUTPUT_EPSG, vbet_network_path)
     project.add_project_vector(proj_nodes['Intermediates'], LayerTypes['VBET_NETWORK'])
 
-    # If bankfull width field exists, buffer by this
-
+    # Get raster resolution as min buffer and apply bankfull width buffer to reaches
+    with rasterio.open(proj_slope) as raster:
+        t = raster.transform
+        min_buffer = (t[0] + (-t[4])) / 2
```

Comment:
1. The slope raster and polygon feature class should be in the same spatial reference. But if they are not, then the raster cell size value that this returns will be in different units than the polygons that are about to be buffered.